plugins/item_licenses/server/constants.py

```diff
@@ -0,0 +1,98 @@
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+
+class PluginSettings:
+    LICENSES = 'item_licenses.licenses'
+
+
+class PluginSettingsDefaults:
+    defaults = {
+        PluginSettings.LICENSES: [
+            {
+                # "Popular" licenses from
+                # https://opensource.org/licenses/category
+                'category': 'Code Licenses',
+                'licenses': [
+                    {
+                        'name': 'Apache License 2.0'
+                        },
+                    {
+                        'name': 'BSD 3-Clause "New" or "Revised" License'
+                        },
+                    {
+                        'name': 'BSD 2-Clause "Simplified" or "FreeBSD" License'
+                        },
+                    {
+                        'name': 'GNU General Public License (GPL) 2.0'
+                        },
+                    {
+                        'name': 'GNU General Public License (GPL) 3.0'
+                        },
+                    {
+                        'name': 'GNU Lesser General Public License (LGPL) 2.1'
+                        },
+                    {
+                        'name': 'GNU Lesser General Public License (LGPL) 3.0'
+                        },
+                    {
+                        'name': 'MIT License'
+                        },
+                    {
+                        'name': 'Mozilla Public License 2.0'
+                        }
+                    ]
+                },
+            {
+                # See http://creativecommons.org/licenses/
+                'category': 'Content Licenses',
+                'licenses': [
+                    {
+                        'name': 'CC0'
+                        },
+                    {
+                        'name': 'CC Attribution (BY)'
+                        },
+                    {
+                        'name': 'CC Attribution-ShareAlike (BY-SA)'
+                        },
+                    {
+                        'name': 'CC Attribution-NoDerivs (BY-ND)'
+                        },
+                    {
+                        'name': 'CC Attribution-NonCommercial (BY-NC)'
+                        },
+                    {
+                        'name': 'CC Attribution-NonCommercial-ShareAlike '
+                        '(BY-NC-SA)'
+                        },
+                    {
+                        'name': 'CC Attribution-NonCommercial-NoDerivs '
+                        '(BY-NC-ND)'
+                        },
+                    {
+                        'name': 'Public Domain'
```

Comment:
I wonder if we should try to name this differently to prevent people from misusing it. In the US, you cannot affirmatively just choose to place something into the public domain. I'm worried people will say "I want to share this with everyone" and chose this license, when in fact, it provides no legal basis for sharing.